uplinks/freifunk-berlin-uplink-no-tunnel/uci-defaults/freifunk-berlin-z95_notunnel

```diff
@@ -38,13 +35,4 @@ uci set network.ffuplink_dev.macaddr=$macaddr
 uci set network.wan.ifname="$(uci get network.wan.ifname) ffuplink_wan"
 uci commit network.ffuplink_dev
 
-uci delete network.ffuplink
-uci set network.ffuplink=interface
-uci set network.ffuplink.ifname=ffuplink
-uci set network.ffuplink.proto=dhcp
-# Put the resulting routing information from the dhcp request in the ffuplink table
-# instead of the default table.  This prevents the dhcp request for the ffuplink
-# interface from overwriting the routing table entries needed by br_wan.
-uci set network.ffuplink.ip4table=ffuplink
-uci set network.ffuplink.ip6table=ffuplink
-uci commit network
+create_ffuplink
```

Comment:
proto needs to be set to dhcp after calling create_ffuplink, followed by another uci commit network.